Don't raise error if version.cmake is not present, which happens when installing from release tarball
Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
diff -ur davix-R_0_7_5.orig/CMakeLists.txt davix-R_0_7_5/CMakeLists.txt
--- davix-R_0_7_5.orig/CMakeLists.txt	2019-08-28 10:43:48.000000000 +0200
+++ davix-R_0_7_5/CMakeLists.txt	2020-04-01 18:39:17.024496680 +0200
@@ -40,7 +40,7 @@
 # A bit hacky.
 #-------------------------------------------------------------------------------
 include(${CMAKE_CURRENT_SOURCE_DIR}/release.cmake REQUIRED)
-include(${CMAKE_CURRENT_SOURCE_DIR}/version.cmake)
+include(${CMAKE_CURRENT_SOURCE_DIR}/version.cmake OPTIONAL)
 message("Configuring cmake for davix version: ${VERSION_FULL}")
 
 set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules/")
